packages/react/src/components/_InputWrapper/InputWrapper.tsx

```diff
@@ -201,15 +202,18 @@ const CharacterCounter = ({
       >
         {srLabel}
       </span>
-      <span
-        className={[
-          classes.characterLimitLabel,
-          hasExceededLimit ? classes.characterLimitExceeded : '',
-        ].join(' ')}
-        aria-live={hasExceededLimit ? 'polite' : 'off'}
-      >
-        {label(currentCount)}
-      </span>
+      <div className={classes.characterLimitLabel}>
```

Comment:
This should also be a `span` element since `CharacterCounter` is rendered inside another `span`.

Reply:
Fixed! :)